### ISSUE TITLE
Create a card with nothing else than a joined image

### ISSUE THREAD

**Issue by Kalytis** [TheLastProject/Catima#676]
Right now, adding an image to a card is a complement to a generated QR code. But for some cards, it would be nice to have an option to create a card that just contains the image from the gallery. No card number, no QR code, just the image.

i.e, went to Disneyland Paris, wanted to add the ticket to Catima, but the Catima card with the generated QR code did not contain the date of the ticket (which was displayed on the image), so I couldn't use Catima.

We could also think about a cropping system for images, which would allow to take a screenshot of something we want to add to Catima, add it via sharing (#614), then cropping it into the app to a fitting format.

Thanks by advance !

**Comment by TheLastProject** [comment]
So, "this card has no barcode" is not enough for your use case? Your ticket does not have ANY kind of ID whatsoever? That seems quite odd to me

**Comment by Kalytis** [comment]
It could work, even though when I use "Add from gallery", it does not add the image to the card, I have to add it again during the edition of the card's details.

**Comment by Kalytis** [comment]
Also, is there a way to fullscreen the image/code during card visualisation ? Sometimes the upper arrow isn't sufficient to show the card at an appropriate size. Clicking the image/code does nothing.

**Comment by TheLastProject** [comment]
> It could work, even though when I use "Add from gallery", it does not add the image to the card, I have to add it again during the edition of the card's details.

That screen is intended to search for a barcode in an image file, not to add an image to a card. It does kinda make sense to make it useful for that way though, it is a logical flow.

**Comment by TheLastProject** [comment]
> Also, is there a way to fullscreen the image/code during card visualisation ? Sometimes the upper arrow isn't sufficient to show the card at an appropriate size. Clicking the image/code does nothing.

Please create 1 GitHub issue per issue, otherwise it becomes really hard to find all requests back. I would need some more info on in what way it isn't sufficient (like an example).

**Comment by TheLastProject** [comment]
If this gets further discussed/worked on, 1057 should be done first for a nicer workflow for this.
